services/filemanager/digitalOceanSpacesManager.go

```diff
@@ -133,8 +140,47 @@ func (manager *DOSpacesManager) ListFilesWithPrefix(prefix string, maxItems int6
 	return
 }
 
-func (manager *DOSpacesManager) DeleteObjects(locations []string) (err error) {
-	return
+func (manager *DOSpacesManager) DeleteObjects(keys []string) (err error) {
+	sess := manager.getSession()
+	if err != nil {
+		return fmt.Errorf(`error starting S3 session: %v`, err)
```

Comment:
mention s3 here might be confusing, also error messages should not contain the word error:
```suggestion
		return fmt.Errorf(`get session: %v`, err)
```

```go
err := fmt.Errorf("establish connection error")

err2 := fmt.Errorf(`error starting S3 session: %v`, err)

if err2 != nil {
    log.Error(err2)
}
```

Would print: `ERROR: error starting S3 session: establish connection error`
if avoid adding error in fmt.Errorf:
`ERROR: starting S3 session: establish connection`